### PR TITLE
Update PSR v0.6.1 #14118

### PR DESCRIPTION
**Pull request by emiliodeg**
Hello!

* Type: update
* Link to issue:  #14118 

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [ ] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:
Changed PSR version from v0.6.0 to v0.6.1
Thanks

### CODE REVIEW

**Comment by niden** [comment]
I had this issue before. Local/nanobox needs 0.6.1 which is the latest version. However when the code goes to Travis it complains about the version and can only work with 0.6.0

@sergeyklay any thoughts on this?

**Comment by zsilbi** [comment]
I changed it to `^0.6` in my local config.

File: composer.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "ext-json": "*",
     "ext-libxml": "*",
     "ext-mbstring": "*",
-    "ext-psr": "0.6.0",
+    "ext-psr": "0.6.1",
```
Comment (sergeyklay) [inline composer.json]:
```suggestion
    "ext-psr": "^0.6",
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**